packages/lesswrong/lib/collections/comments/types.ts

```diff
@@ -1,4 +1,2 @@
-export enum TagCommentType {
-  Subforum = "SUBFORUM",
-  Discussion = "DISCUSSION",
-}
+export type TagCommentType = "SUBFORUM" | "DISCUSSION"
```

Comment:
I changed this from an enum to a literal type because there were some places where converting between enum and string was confusing

Reply:
JP's feeling is that enum's are to be recommended against, in favor of unions of string literals anyway